### ISSUE TITLE
Don't track screen views automatically by default

### ISSUE THREAD

**Issue by brianjd** [charlires/react-native-segment-analytics#19]
I believe this can prove to be problematic depending on the navigation solution being used.

**Comment by idris** [comment]
Yes, [the `setup` method](https://github.com/charlires/react-native-segment-analytics/blob/e421fe2b773def1009b9b34f8c8c45a3704e5f2c/ios/SegmentAnalytics/Classes/SegmentAnalytics.m#L14-L21) should accept a Map/Dictionary of configuration options:
```js
{
  flushAt?: int = 1,
  shouldUseLocationServices?: boolean = true,
  recordScreenViews?: boolean = true,
  trackApplicationLifecycleEvents?: boolean = true,
}
```

**Comment by sunnymopada** [comment]
@charlires Any update on this?

**Comment by roycclu** [comment]
@SanyasiraoM 
@charlires 
Looks like no response...
For now, I'm going in the SegmentAnalytics.m file and changing line 17 manually:

```
    configuration.recordScreenViews = YES;
```
to 

```
    configuration.recordScreenViews = NO;
```

Then patch-package to keep this configuration on every install. Not great. But does the job reasonably well.
